.github/workflows/pre-commit.yml

```diff
@@ -12,6 +12,8 @@ jobs:
       SKIP: no-commit-to-branch
     steps:
     - uses: actions/checkout@v4
+      with:
+        persist-credentials: false
```

Comment:
Good point, I don't know much about this part. I suppose both answers can be argued: no, we don't use the Github token, therefore we don't need this setting. But also yes, we should use it, as it closes a potential security hole: we don't use the token now, but we may in the future, and we may easily forget to enable this feature, plus it speaks about "credentials", there may be other tokens, not just the GH one.

Despite us not using a Github token today, I for one would add the setting to prevent any issue in the future.

Reply:
Ok, sounds reasonable.